hazelcast/src/main/java/com/hazelcast/multimap/impl/operations/MergeOperation.java

```diff
@@ -62,7 +62,7 @@ public MergeOperation(String name, List<MultiMapMergeContainer> mergeContainers,
 
     @Override
     public void run() throws Exception {
-        MultiMapContainer container = getOrCreateContainer();
+        MultiMapContainer container = getOrCreateContainerWithoutAccess();
```

Comment:
We cannot update the `lastAccessTime` of the local container here, otherwise the `LastAccessMergePolicy` won't work properly.